pipeline/pipeline/calculate_metrics_file.py

```diff
@@ -66,6 +81,14 @@ def calculate_metrics(
     # print(drop_rows.index[0:5])
     df.drop(drop_rows.index, axis=0, inplace=True)
 
+    # create non imputed temp column before it gets imputed.Temp column is created just to store values
+    # It will be deleted and values transferred to another column to not affect the position of columns
+    df["delta.tested.non.impu.temp"] = df["delta.tested"].fillna(value=0)
+    df["delta.confirmed.non.impu.temp"] = df["delta.confirmed"].fillna(value=0)
+    df["delta.deceased.non.impu.temp"] = df["delta.deceased"].fillna(value=0)
+    df["delta.recovered.non.impu.temp"] = df["delta.recovered"].fillna(value=0)
+    df["delta.other.non.impu.temp"] = df["delta.other"].fillna(value=0)
```

Comment:
why a temp column? I don't see any mutations in the .non.impu column...

I would prefer more pythonic names as well. .not_imputed (or .raw?)
But we're working here with legacy code so don't think to hard about naming. but full words above abbreviations is better if it's managable.

Reply:
Temp column is created to hold the values before it is imputed. Since we cannot change the order of columns (as discussed in the meeting on sunday), I have reassigned these values to new columns once the original ones are created.

Regarding the names, I have been following the naming conventions already in use. We can look into changing it in future once the full review is done.